docs/examples/table/sortableTableCells.js

```diff
@@ -53,7 +53,9 @@ export default function SortableHeaderExample(): Node {
             <Text>Snax</Text>
           </Table.Cell>
           <Table.Cell>
-            <Text align="end">$50</Text>
+            <Box width={100}>
```

Comment:
instead of doing width={100} ca we use 

```
        <colgroup>
          <col span="1" style={{ width: '60%' }} />
          <col span="1" style={{ width: '15%' }} />
          <col span="1" style={{ width: '15%' }} />
          <col span="1" style={{ width: '15%' }} />
        </colgroup>

```